libopenage/renderer/resources/parser/parse_blendtable.cpp

```diff
@@ -67,16 +67,16 @@ PatternData parse_pattern(const std::vector<std::string> &args) {
 	return pattern;
 }
 
-BlendTableInfo parse_blendtable_file(const util::Path &file,
+BlendTableInfo parse_blendtable_file(const util::Path &path,
                                      const std::shared_ptr<AssetCache> &cache) {
-	if (not file.is_file()) [[unlikely]] {
+	if (not path.is_file()) [[unlikely]] {
 		throw Error(MSG(err) << "Reading .bltable file '"
-		                     << file.get_name()
+		                     << path.get_name()
 		                     << "' failed. Reason: File not found");
 	}
 
-	auto content = file.open();
-	auto lines = content.get_lines();
```

Comment:
we should improve the get_lines function to improve this for all read files?

Reply:
I've modified it to use the new `util::split_newline` function now.